test/feature/ProtectedReleases.feature

```diff
@@ -1,33 +1,49 @@
 Feature: protected releases
```

Comment:
@kereid @xhoenner @pblain Katherine FYI

This feature spec has been updated. In summary, the things that have been changed are:
* split the protection notification into 2 scenarios (for clarity)
* no protection notification when creating a release (not associated with project, yet)
* other small tidy-ups

It's up to you guys to decide whether we need to notify the data committee of these changes.